app/controllers/api/v1/errors_controller.rb

```diff
@@ -6,6 +6,16 @@ def index
   end
 
   def create
+    @subscriber = Subscriber.where("email = ? and website_id = ?", params['email'], @current_site.id)
+    @error = Issue.where("description = ?", params['message'])
```

Comment:
asta nu prea are sens. description ala cel mai probabil o sa fie un backtrace sau un mesaj pe care il trimite siteu cu eroarea. Ti-am zis ca params['message'] asta nu prea are loc in momentul asta. Ce putem sa facem e sa adaugam un nou tabel cu mesajele. Si sa fie mesajele legate de issue_id si subscriber_id